### DISCUSSION TITLE
Suggestion for PLPv3b

### DISCUSSION THREAD

**Discussion by anonymous14725** [PLPlanguage/PLPv2b#13] [General]
Suggestion for PLPv3b

**Comment by barry-far** [comment]
Hi @anonymous14725 , thanks for invitation to your project. i just wonder to know why? :) can i help ?

**Reply by anonymous14725** [reply]
Hi @barry-far 
The goal is to gather new developers for the PLP programming language,
I wanted you to join this project and invite some people who are developers to this project if you need to.
Purpose of the PLPv3b 

- [ ] 1. Work almost on Unix-like operating systems, Windows, MacOS and Plan9 compiler.
- [ ] 2. The compiler should be dynamically updated.
- [ ] 3. In graphical operating systems such as Windows, MacOS and Linux Compiler GUI works.

These three things require programmers who are full of ideas.

**Reply by barry-far** [reply]
> Hi @barry-far The goal is to gather new developers for the PLP programming language, I wanted you to join this project and invite some people who are developers to this project if you need to. Purpose of the PLPv3b
> 
> * [ ]  1. Work almost on Unix-like operating systems, Windows, MacOS and Plan9 compiler.
> * [ ]  2. The compiler should be dynamically updated.
> * [ ]  3. In graphical operating systems such as Windows, MacOS and Linux Compiler GUI works.
> 
> These three things require programmers who are full of ideas.

awesome , i suggest you to create a post & tag me in X (formely Twitter).
my X ID is : @fard1627

Unfortunately mthis programming language is out of my expertise. ❤️

**Reply by anonymous14725** [reply]
thank you, If you know someone who is active in this field, introduce me
PLP language @barry-far
